client/app/config/index.js

```diff
@@ -46,8 +46,42 @@ const requirements = [
   ngGridster.name,
 ];
 
+const dashboardGridOptions = {
+  columns: 6,
+  pushing: true,
+  floating: true,
+  swapping: true,
+  width: 'auto',
+  colWidth: 'auto',
+  rowHeight: 100,
+  margins: [0, 0],
+  outerMargin: false,
+  sparse: false,
+  isMobile: false,
+  mobileBreakPoint: 1200,
```

Comment:
We need to change this to be lower. I will play around with a few values to see what works better.

Reply:
Current version has a breakpoint at Bootstraps `-lg-` - `1200px`, so it's just kept as-is.
I think, the best decision it to make it adjustable (via dashboard options) - because the best breakpoint value depends on dashboard layout and configuration of widgets. How do you this - does it make any reason?